src/pages/projects/index.tsx

```diff
@@ -5,31 +5,166 @@ import { getProjectsData, getSiteData } from "../../lib/api";
 import { IProjectsData } from "../../interfaces/project";
 import Image from "next/image";
 import Link from "next/link";
+import styled from "@emotion/styled";
+import { IconButton } from "../../components/global/icon-button";
+import { useEffect, useState } from "react";
 
 interface IIndex {
   siteData: ISiteData;
   pageData: IProjectsData;
   preview: boolean;
 }
 
+interface IStyledContainerProps {
+  imageSrc: string;
+}
+
+const StyledContainer = styled.div<IStyledContainerProps & { isDesktop: boolean }>`
+  display: flex;
+  flex-direction: column;
+  background-image: url(${(props) => props.imageSrc});
+  background-size: ${(props) => (props.isDesktop ? "cover" : "cover")};
+  background-repeat: no-repeat;
+  background-position: center;
+  height: ${(props) => (props.isDesktop ? "12rem" : "20rem")};
```

Comment:
Now increase to 20rem for desktop (440px)

I'm mostly working with rem instead of px, but let me know if you want me to change everything to px to perfectly match the design. I just feel rem is nicer if the user zoom in using the browser.